### PR TITLE
Update to kickoff release

### PR DESCRIPTION
**Pull request by outsidermm**

### CODE REVIEW

**Comment by outsidermm** [comment]
should the python version in the pipfile be 3.12

**Comment by outsidermm** [comment]
why is ci not happy even when i changed the commands

**Comment by auscompgeek** [comment]
Have you read the logs?

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 hypothesis
 pytest
-pyfrc==2024.0.0b1
-robotpy-installer==2024.0.0b3
+pyfrc==2024.0.1
+robotpy-installer==2024.0.2
```
Comment (auscompgeek) [inline requirements-dev.txt:4]: Is there a reason to continue pinning these?

**Comment by outsidermm** [comment]
is the correct command to test, sim and deploy `python -m robotpy [sim/test/deploy]`?
Because my mac command is `python3 -m robot [sim/test/deploy]`

**Comment by auscompgeek** [comment]
> is the correct command to test, sim and deploy `python -m robotpy [sim/test/deploy]`?

yes

> Because my mac command is `python3 -m robot [sim/test/deploy]`

this indicates you haven't installed the dependency versions this PR is updating everything to

File: Pipfile [hunk 2]
```diff
@@ -21,6 +20,6 @@ pytest = ">=7.2.0"
 python_version = "3.11"
 
 [scripts]
-deploy = "python robot.py deploy"
-sim = "python robot.py sim"
-test = "python robot.py test"
+deploy = "python -m robot deploy"
+sim = "python -m robot sim"
+test = "python -m robot test"
```
Comment (auscompgeek) [inline Pipfile:25]: Do these scripts work?

Comment (outsidermm) [reply Pipfile:25]: oops

**Review by auscompgeek — APPROVED** [review]

**Pull request merged**